back/src/Services/SocketManager.ts

```diff
@@ -1461,6 +1466,90 @@ export class SocketManager {
         }
     }
 
+    handleKickSpaceUserMessage(pusher: SpacesWatcher, kickUserMessage: KickOffSpaceUserMessage) {
+        const space = this.spaces.get(kickUserMessage.spaceName);
+        if (!space) return;
+        pusher.write({
+            message: {
+                $case: "kickOffSpaceUserMessage",
+                kickOffSpaceUserMessage: {
+                    spaceName: kickUserMessage.spaceName,
+                    userId: kickUserMessage.userId,
+                    filterName: kickUserMessage.filterName,
+                },
+            },
+        });
+    }
+
+    handleMuteMicrophoneSpaceUserMessage(
```

Comment:
All the methods below do not apply to "spaces". They target groups.
So there should be no "space" in the method names of this class, right?

Reply:
In this case is a SpacesWatcher passed in parameter, so we use space.